src/main/java/org/apache/sysds/runtime/controlprogram/federated/FederationMap.java

```diff
@@ -50,6 +50,7 @@
 		ROW, // row partitioned, groups of rows
 		COL, // column partitioned, groups of columns
 		FULL, // Meaning both Row and Column indicating a single federated location and a full matrix
+		PART, // Partial aggregates in several federated locations with addition as the aggregation method
```

Comment:
Call it Overlap, to not confuse (at least me) and this overlap is a concept i have in compression as well, where i call it ... overlap.

Reply:
I chose this name because it matches the other names: it is short and in contrast to "FULL".